packages/rainbowkit/src/wallets/walletConnectors/frontierWallet/frontierWallet.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface FrontierWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const frontierWallet = ({
+  chains,
+  shimDisconnect,
```

Comment:
Reference recent changes to connectors for `options` pass-through https://github.com/rainbow-me/rainbowkit/blob/0d5640929326d65673596a11cd018a9c6524ff8c/packages/rainbowkit/src/wallets/walletConnectors/metaMaskWallet/metaMaskWallet.ts#L31-L34